examples/complete/variables.tf

```diff
@@ -31,3 +31,16 @@ variable "glb_name" {
   type        = string
   default     = "glb"
 }
+
+variable "enabled_rulesets" {
+  description = "List of rulesets and whether they are enabled or not"
```

Comment:
Can we add more info like a link from where user can check the rulesets information and accordingly define the variable